samples/ControlCatalog.Android/SplashActivity.cs

```diff
@@ -1,6 +1,13 @@
 ﻿using Android.App;
 using Android.Content;
 using Android.OS;
+<<<<<<< HEAD
+=======
+using Application = Android.App.Application;
+
+using Avalonia;
+using Avalonia.Android;
+>>>>>>> 0f6e65aa4 (android support)
```

Comment:
Pending merge conflict

Reply:
Removed. Wonder why git didn't warn me of this